chains/evm/contracts/ccvs/CommitRamp.sol

```diff
@@ -38,16 +44,24 @@ contract CommitOnRamp is Ownable2StepMsgSender, BaseOnRamp {
     _setDynamicConfig(dynamicConfig);
   }
 
+  function supportsInterface(
+    bytes4 interfaceId
+  ) external pure returns (bool) {
+    return interfaceId == type(ICCVOffRampV1).interfaceId || interfaceId == type(IERC165).interfaceId;
+  }
+
   /// @notice Forwards a message from CCV proxy to this verifier for processing and returns verifier-specific data.
   /// @dev This function is called by the CCV proxy to delegate message verification to this specific verifier.
   /// It performs critical validation to ensure message integrity and proper sequencing.
+  /// @param originalCaller The original caller of forwardToVerifier, which is passed as input to enable proxy patterns.
   /// @param message Decoded MessageV1 payload for verification.
-  /// @param 2nd parameter is messageId, unused. Allows the onRamp to use the messageId to enable their offchain components.
-  /// @param 3rd parameter is feeToken, unused. Helps determine verifierReturnData.
-  /// @param 4th parameter is feeTokenAmount, unused. Helps determine verifierReturnData.
-  /// @param 5th parameter, is verifierArgs (opaque verifier-specific args), unused. Helps determine verifierReturnData.
+  /// @param 3rd parameter is messageId, unused. Allows the onRamp to use the messageId to enable their offchain components.
```

Comment:
Let's not introduce the pattern of enumerating params. We don't do that anywhere and here you can clearly see why

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/8f1762ef84e1348c20ab970fbc61f97ba2327bf7